warehouse/templates/packaging/detail.html

```diff
@@ -244,7 +244,7 @@ <h2 class="no-top-padding">Reverse Dependencies (TODO)</h2>
             <div class="release{{ ' release--latest' if loop.first else ' release--oldest' if loop.last }}{{ ' release--current' if is_current }}">
               <div class="release__meta">
                 {% if is_current %}
-                <p class="release__version">{{ hrelease.version }}</p>
+                <p class="release__version"><a href="{{ request.route_path('packaging.release', name=release.project.name, version=hrelease.version) }}">{{ hrelease.version }}</a></p>
```

Comment:
It looks like this change makes this line exactly equal to the line in the else clause of this if statement. Perhaps it'd make more sense to just move this line out of the if statement all together, and make the if statement just ``{% if is_current %}<span class="badge">This version</span>{% endif %}``, unless I'm missing a difference.